fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -35,6 +35,14 @@ def __init__(self, name, init_config, instances):
 
         self._fluentd_command = self.instance.get('fluentd', init_config.get('fluentd', 'fluentd'))
 
+        self.url = self.instance.get('monitor_agent_url')
+        parsed_url = urlparse(self.url)
+        self.monitor_agent_host = parsed_url.hostname
+        self.monitor_agent_port = parsed_url.port or 24220
```

Comment:
(nit) Might be good to have the default port as constant `DEFAULT_PORT = 24220`.